src/main/resources/application.yaml

```diff
@@ -172,6 +172,7 @@ scheduling:
     upload-documents:
       delay: ${UPLOAD_TASK_DELAY} # In milliseconds
       enabled: ${UPLOAD_TASK_ENABLED}
+      max_tries: ${UPLOAD_MAX_TRIES}
```

Comment:
perhaps `max-retries` makes more sense does it not?

Reply:
Well, it depends.
Currently this value is set to 5. I assume the intent was to try 5 times.
Not fail once and then retry 5 times, ie 6 attempts in total.

Reply:
Not sure if max tries is the correct word ? may be max_upload_attempts?